Patches/Vanilla Apparel Expanded/ThingDefs_Misc/Apparel_Industrial.xml

```diff
@@ -1,121 +1,176 @@
-				</value>
-			</li>
+				<!-- == VAE_Apparel_ChefsUniform == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_ChefsUniform"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>1.2</StuffEffectMultiplierArmor>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_ChefsUniform == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_ChefsUniform"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<StuffEffectMultiplierArmor>1.2</StuffEffectMultiplierArmor>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_MilitaryJacket == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_MilitaryJacket"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<Bulk>5</Bulk>
+						<WornBulk>1</WornBulk>
+						<StuffEffectMultiplierArmor>6</StuffEffectMultiplierArmor>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_MilitaryJacket == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_MilitaryJacket"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<Bulk>5</Bulk>
-					<WornBulk>1</WornBulk>
-					<StuffEffectMultiplierArmor>6</StuffEffectMultiplierArmor>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_MilitaryUniform == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_MilitaryUniform"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<Bulk>2</Bulk>
+						<WornBulk>0.5</WornBulk>
+						<StuffEffectMultiplierArmor>1.4</StuffEffectMultiplierArmor>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_MilitaryUniform == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_MilitaryUniform"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<Bulk>2</Bulk>
-					<WornBulk>0.5</WornBulk>
-					<StuffEffectMultiplierArmor>1.4</StuffEffectMultiplierArmor>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_BuildersJacket == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_BuildersJacket"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<Bulk>5</Bulk>
+						<WornBulk>1</WornBulk>
+						<ArmorRating_Sharp>0.55</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_BuildersJacket"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>0.4</ArmorRating_Blunt>
+					</value>
+				</li>
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_BuildersJacket"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>20</Cloth>
```

Comment:
Are there additional materials required along with the cloth? The base cost of the normal jacket is 60, and I think the builder's jacket (a heavier, slightly more rugged jacket) should cost a but more.